### PR TITLE
add function to calcuate volume of a cylinder in js, go, php, py, c, …

### LINKED ISSUE #4032: Write a C program to find volume of a cylinder
### Description

Write a C program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.c` file
- Add `FindVolumeOfACylinder.c` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #4033: Write a C# program to find volume of a cylinder
### Description

Write a C# program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.cs` file
- Add `FindVolumeOfACylinder.cs` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #4034: Write a C++ program to find volume of a cylinder
### Description

Write a C++ program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.cpp` file
- Add `FindVolumeOfACylinder.cpp` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #4037: Write a GO program to find volume of a cylinder
### Description

Write a GO program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `find_volume_of_a_cylinder.go` file
- Add `find_volume_of_a_cylinder.go` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #4039: Write a Java program to find volume of a cylinder
### Description

Write a Java program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.java` file
- Add `FindVolumeOfACylinder.java` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #4040: Write a JavaScript program to find volume of a cylinder
### Description

Write a JavaScript program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.js` file
- Add `FindVolumeOfACylinder.js` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #4044: Write a PHP program to find volume of a cylinder
### Description

Write a PHP program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `find_volume_of_a_cylinder.php` file
- Add `find_volume_of_a_cylinder.php` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #4045: Write a Python program to find volume of a cylinder
### Description

Write a Python program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `find-volume-of-a-cylinder.py` file
- Add `find-volume-of-a-cylinder.py` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #4047: Write a Ruby program to find volume of a cylinder
### Description

Write a Ruby program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.rb` file
- Add `FindVolumeOfACylinder.rb` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #4049: Write a Scala program to find volume of a cylinder
### Description

Write a Scala program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `findVolumeOfACylinder.scala` file
- Add `findVolumeOfACylinder.scala` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #5382: Write a C program to find volume of a cylinder
### Description

Write a C program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.c` file
- Add `FindVolumeOfACylinder.c` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #5383: Write a C# program to find volume of a cylinder
### Description

Write a C# program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.cs` file
- Add `FindVolumeOfACylinder.cs` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #5384: Write a C++ program to find volume of a cylinder
### Description

Write a C++ program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.cpp` file
- Add `FindVolumeOfACylinder.cpp` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #5387: Write a GO program to find volume of a cylinder
### Description

Write a GO program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `find_volume_of_a_cylinder.go` file
- Add `find_volume_of_a_cylinder.go` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #5389: Write a Java program to find volume of a cylinder
### Description

Write a Java program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.java` file
- Add `FindVolumeOfACylinder.java` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #5390: Write a JavaScript program to find volume of a cylinder
### Description

Write a JavaScript program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.js` file
- Add `FindVolumeOfACylinder.js` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #5394: Write a PHP program to find volume of a cylinder
### Description

Write a PHP program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `find_volume_of_a_cylinder.php` file
- Add `find_volume_of_a_cylinder.php` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #5395: Write a Python program to find volume of a cylinder
### Description

Write a Python program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `find-volume-of-a-cylinder.py` file
- Add `find-volume-of-a-cylinder.py` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #5397: Write a Ruby program to find volume of a cylinder
### Description

Write a Ruby program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.rb` file
- Add `FindVolumeOfACylinder.rb` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #5399: Write a Scala program to find volume of a cylinder
### Description

Write a Scala program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `findVolumeOfACylinder.scala` file
- Add `findVolumeOfACylinder.scala` file in `program/find-volume-of-a-cylinder` folder

</details>

### LINKED ISSUE #5401: Write a TypeScript program to find volume of a cylinder
### Description

Write a TypeScript program to find volume of a cylinder

Volume of cylinder = π × `Radius`^2 × `Height`

```
Radius : 2
Height : 3

Volume : 37.69911184307752
```
<details>
<summary>How to contribute</summary>

- Save the solution in `FindVolumeOfACylinder.ts` file
- Add `FindVolumeOfACylinder.ts` file in `program/find-volume-of-a-cylinder` folder

</details>

### PR DESCRIPTION
**Pull request by Edward-Regalado**
### Why:

Closes codinasion/codinasion#4032
Closes codinasion/codinasion#4033 
Closes codinasion/codinasion#4034 
Closes codinasion/codinasion#4039 
Closes codinasion/codinasion#4045 
Closes codinasion/codinasion#4047 
Closes codinasion/codinasion#4049 
Closes codinasion/codinasion#4037
Closes codinasion/codinasion#4044
Closes codinasion/codinasion#4040 
Closes codinasion/codinasion#4051

### CODE REVIEW

**Review by anandfresh — CHANGES_REQUESTED** [review]
Can you add the issue id's for all the 11 different language files .
Currently you have attached only for Javascript

**Comment by harshraj8843** [comment]
Hey @Edward-Regalado 

Please comment `!assign` on all the linked issue, to be auto-assigned, to prevent any conflict 👍🏻

**Review by anandfresh — CHANGES_REQUESTED** [review]
@Edward-Regalado  Can you pls make sure you have assigned to the issues you have mentioned . 
I'm not seeing all the issues are assigned to you . Currently you have assigned only the Javascript issue under your name.
do comment !assign on all the linked issue, so that it will get auto assign to you 
Thumb rule : Assign your issues before creating a PR

**Comment by Edward-Regalado** [comment]
@anandfresh My apologies. I went ahead and assigned myself on all linked issues from above.

File: program/find-volume-of-a-cylinder/volumeOfCylinder.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Function to calculate the volume of a cylinder
+function volumeOfCylinder(radius, height) {
+    // Calculate the volume using the formula:
+    // V = π * r^2 * h
+    const volume = Math.PI * radius * radius * height;
+  
+    // Return the calculated volume
+    return volume;
```
Comment (anandfresh) [inline program/find-volume-of-a-cylinder/volumeOfCylinder.js:8]: can we round of the volume result ?

File: program/find-volume-of-a-cylinder/find_volume_of_a_cylinder.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+// Function to calculate the volume of a cylinder
+function volumeOfCylinder($radius, $height) {
+  // Calculate the volume using the formula:
+  // V = π * r^2 * h
+  $volume = M_PI * pow($radius, 2) * $height;
+
+  // Return the calculated volume
+  return $volume;
```
Comment (anandfresh) [inline program/find-volume-of-a-cylinder/find_volume_of_a_cylinder.php:10]: round of the volume

File: program/find-volume-of-a-cylinder/find_volume_of_a_cylinder.go [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import "fmt"
+
+// Function to calculate the volume of a cylinder
+func volumeOfCylinder(radius float64, height float64) float64 {
+  // Calculate the volume using the formula:
+  // V = π * r^2 * h
+  volume := math.Pi * math.Pow(radius, 2) * height
+
+  // Return the calculated volume
+  return volume
```
Comment (anandfresh) [inline program/find-volume-of-a-cylinder/find_volume_of_a_cylinder.go:12]: round of the volume

File: program/find-volume-of-a-cylinder/findVolumeOfACylinder.scala [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// Function to calculate the volume of a cylinder
+def volumeOfCylinder(radius: Double, height: Double): Double = {
+  // Calculate the volume using the formula:
+  // V = π * r^2 * h
+  val volume = math.Pi * radius * radius * height
+
+  // Return the calculated volume
+  volume
```
Comment (anandfresh) [inline program/find-volume-of-a-cylinder/findVolumeOfACylinder.scala:8]: round of the volume to two decimals

File: program/find-volume-of-a-cylinder/findVolumeOfACylinder.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Function to calculate the volume of a cylinder
+def volume_of_cylinder(radius, height)
+    # Calculate the volume using the formula:
+    # V = π * r^2 * h
+    volume = Math::PI * radius * radius * height
+  
+    # Return the calculated volume
+    volume
```
Comment (anandfresh) [inline program/find-volume-of-a-cylinder/findVolumeOfACylinder.rb:8]: round of the volume to two decimals

File: program/find-volume-of-a-cylinder/find-volume-of-a-cylinder.py [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# Function to calculate the volume of a cylinder
+def volume_of_cylinder(radius, height):
+  # Calculate the volume using the formula:
+  # V = π * r^2 * h
+  volume = 3.1415 * radius * radius * height
+
+  # Return the calculated volume
+  return volume
```
Comment (anandfresh) [inline program/find-volume-of-a-cylinder/find-volume-of-a-cylinder.py:8]: round of the volume to two decimals

File: program/find-volume-of-a-cylinder/FindVolumeOfACylinder.ts [hunk 7]
```diff
@@ -0,0 +1,17 @@
+// Function to calculate the volume of a cylinder
+function volumeOfCylinder(radius: number, height: number): number {
+    // Calculate the volume using the formula:
+    // V = π * r^2 * h
+    const volume = Math.PI * radius * radius * height;
+  
+    // Return the calculated volume
+    return volume;
+  }
```
Comment (anandfresh) [inline program/find-volume-of-a-cylinder/FindVolumeOfACylinder.ts:9]: round of the volume to two decimals

File: program/find-volume-of-a-cylinder/FindVolumeOfACylinder.java [hunk 8]
```diff
@@ -0,0 +1,21 @@
+public class VolumeOfCylinder {
+    // Function to calculate the volume of a cylinder
+    public static double volumeOfCylinder(double radius, double height) {
+        // Calculate the volume using the formula:
+        // V = π * r^2 * h
+        double volume = Math.PI * Math.pow(radius, 2) * height;
+
+        // Return the calculated volume
+        return volume;
```
Comment (anandfresh) [inline program/find-volume-of-a-cylinder/FindVolumeOfACylinder.java:9]: round of the volume to two decimals

File: program/find-volume-of-a-cylinder/FindVolumeOfACylinder.cs [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+    // Function to calculate the volume of a cylinder
+    static double VolumeOfCylinder(double radius, double height)
+    {
+        // Calculate the volume using the formula:
+        // V = π * r^2 * h
+        double volume = Math.PI * Math.Pow(radius, 2) * height;
+
+        // Return the calculated volume
+        return volume;
+    }
```
Comment (anandfresh) [inline program/find-volume-of-a-cylinder/FindVolumeOfACylinder.cs:14]: round of the volume to two decimals

File: program/find-volume-of-a-cylinder/FindVolumeOfACylinder.cpp [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+// Function to calculate the volume of a cylinder
+double volumeOfCylinder(double radius, double height) {
+  // Calculate the volume using the formula:
+  // V = π * r^2 * h
+  double volume = M_PI * pow(radius, 2) * height;
+
+  // Return the calculated volume
+  return volume;
+}
```
Comment (anandfresh) [inline program/find-volume-of-a-cylinder/FindVolumeOfACylinder.cpp:12]: round of the volume to two decimals

File: program/find-volume-of-a-cylinder/FindVolumeOfACylinder.c [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+// Function to calculate the volume of a cylinder
+double volumeOfCylinder(double radius, double height) {
+  // Calculate the volume using the formula:
+  // V = π * r^2 * h
+  double volume = M_PI * pow(radius, 2) * height;
+
+  // Return the calculated volume
+  return volume;
+}
```
Comment (anandfresh) [inline program/find-volume-of-a-cylinder/FindVolumeOfACylinder.c:12]: round of the volume to two decimals

**Review by anandfresh — CHANGES_REQUESTED** [review]

**Review by harshraj8843 — APPROVED** [review]
Thanks for contributing and sorry for the delay 🤗

**Pull request merged**